libraries/chain/include/eosio/chain/block_timestamp.hpp

```diff
@@ -17,7 +17,10 @@ namespace eosio { namespace chain {
    template<uint16_t IntervalMs, uint64_t EpochMs>
    class block_timestamp {
       public:
-         explicit block_timestamp( uint32_t s=0 ) :slot(s){}
+
+         block_timestamp() : slot(0) {}
+
+         explicit block_timestamp( uint32_t s ) :slot(s){}
```

Comment:
Why this change? What difference in behavior does this provide?

Reply:
The default constructor is not `explicit` anymore, so structs containing `block_timestamp` can be constructed as aggregates.